### PR TITLE
Basic personal page with email, first name & last name

### PR DESCRIPTION
**Pull request by YCHQR**
Did a basic personal page with email, first name, last name and contact. Because behind schedule I will adjust after.

### CODE REVIEW

**Comment by YCHQR** [comment]
You can find it at /account

File: app/views/pages/account.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+<div class="card">
+  <h1><%= current_user.first_name %> <%= current_user.last_name %></h1>
+  <p>Email: <%= current_user.email %></p>
+  <a href="#"><i class="fa fa-twitter"></i></a>
+  <a href="#"><i class="fa fa-linkedin"></i></a>
+  <a href="#"><i class="fa fa-facebook"></i></a>
+  <p><button>Contact</button></p>
```
Comment (piranha73) [inline app/views/pages/account.html.erb:9]: I know that this are really cool but not linking to anywhere, so comment it. Instead add an email logo linking to mailto:current_user.email

File: app/controllers/pages_controller.rb [hunk 2]
```diff
@@ -3,4 +3,13 @@ class PagesController < ApplicationController
...
   end
+
+  def account
+  end
+
+  private
+
+  def account_params
+    params.require(:product).permit(:email, :first_name, :last_name, :address)
+  end
```
Comment (piranha73) [inline app/controllers/pages_controller.rb:14]: Why do we need this params.require method?

**Review by piranha73 — COMMENTED** [review]
It's all good, I need just a "front-end" change and to understand what are you doing with params in Pages Contoroller

**Comment by YCHQR** [comment]
Resolved the changes in personal page

**Review by piranha73 — APPROVED** [review]

**Pull request merged**